.pre-commit-config.yaml

```diff
@@ -106,3 +106,9 @@ repos:
   rev: 2.5.3
   hooks:
     - id: pdm-lock-check
+#    - id: pdm-export
```

Comment:
@cwegener Have you seen this issue before where the pre-commit hook doesn't appear to use the right version of pdm and then fails?

Reply:
Interesting. No. I haven't come across this before. I'll keep an eye out for it.

Reply:
> Have you seen this issue before where the pre-commit hook doesn't appear to use the right version of pdm and then fails?
Where are you seeing theses issues? Running pre-commit locally or in the GH actions?

Reply:
I just noticed something really strange ... the pdm.lock file schema version is 3.1 .... Which is the lockfile schema from pdm version 2.0 ... there have been multiple lockfile verison upgrades since then.

https://github.com/DataJunction/dj/blob/49ac7b00015d3b9d4abea508d64f02a9da82d48d/pdm.lock#L818

We'll need to fix whatever is broken there first, because somehow that pdm.lock file got created/edited with an old pdm 2.0 version ...

Reply:
@shangyian Somehow you downgraded pdm to 2.0 last week ... :joy: https://github.com/DataJunction/dj/commit/afb1b09#diff-4dc59eeadc4eee1785e5a8461355721763a4ab9a32b8e3b88def661da9cb497cR818

Reply:
I made #539

Reply:
Thanks for the fix! As for the pre-commit hook, I was seeing this only in GH actions, but not while running it locally. In any case it shouldn't be needed with the post hook.